kubediff

```diff
@@ -31,7 +33,7 @@ supply the kubeconfig for the appropriate environment.""")
   if options.json:
     printer = JSONPrinter()
 
-  failed = check_files(args, printer, options.kubeconfig)
+  failed = check_files(args, printer, options.kubeconfig, options.context, options.namespace)
```

Comment:
If that would be your preference, that works.

I may switch the whole thing over to argparse from optparse in the process...